tests/e2e/e2e_test.go

```diff
@@ -1152,10 +1198,103 @@ func TestE2E(t *testing.T) {
 				if err != nil {
 					return err
 				}
-
 				return removeNodeTaint(t, f, taintedNode.Name, taintKey)
 			},
 		},
+		testExecution{
+			Name:       "TestScanSettingBinding",
+			IsParallel: true,
+			TestFn: func(t *testing.T, f *framework.Framework, ctx *framework.Context, mcTctx *mcTestCtx, namespace string) error {
+				objName := getObjNameFromTest(t)
+
+				rhcosPbName := objName + "-rhcos4"
+				err := createReadyProfileBundle(t, f, rhcosPbName, namespace, contentImagePath, rhcosContentFile)
```

Comment:
we already have some default profilebundles, would it be OK to use those? or do you think there's value in creating this temporary one?

Reply:
I used them, btw I ran into an issue that unless we delete the bundles in the e2e test teardown, the test namespace stays stuck in terminating, so I added two defer statements to the helpers.go to address that, is that OK? There is still a helper function that waits until the bundles are ready on test setup.